docs/concepts/resolution.md

```diff
@@ -296,6 +296,12 @@ To ensure reproducibility, messages for unsatisfiable resolutions will not menti
 distributions were excluded due to the `--exclude-newer` flag — newer distributions will be treated
 as if they do not exist.
 
+!!! note
+
+    The `--exclude-newer` option is only applied to registry packages and will not downgrade
```

Comment:
Maybe I should have said "packages read from a registry" instead of "registry packages"?

Reply:
Is that clearer?

Reply:
yes i agree. will update

Reply:
or maybe something about "at resolution time" / "when resolving packages" or similar?

Reply:
Well, we include the local distributions in the resolution / perform resolution with them — we just don't read them from a registry we read their metadata from the local environment.